en/docs/install-and-setup/setup/api-controller/building-jenkins-ci-cd-pipeline-apictl.md

```diff
@@ -0,0 +1,328 @@
+# Building a CI/CD Pipeline for APIs Using Jenkins
+
+In the modern digital-era, many organizations adopt digital transformation technologies to keep up with the changes. APIs are one of the basic building blocks used in a digitally-driven organization. When the number of APIs managed by the organization grows, they need to have an automated process to handle the rapid API development process. Having a proper continuous integration and continuous deployment (CI/CD) process would give an added advantage to your organization. 
+
+In this section, you will learn how to build an automated process using the WSO2 API Controller(CTL) with WSO2 API Manager 4.0.0 . You will see how a solution is built using version control system (Github), artifact repository(JFrog Artifactory) and CI/CD Tool (Jenkins).
```

Comment:
```suggestion
In this section, you will learn how to build an automated process using the APICTL with WSO2 API-M 4.0.0 . You will see how a solution is built using a version control system (Github), artifact repository (JFrog Artifactory) and CI/CD Tool (Jenkins).
```
Let's use the abbreviations apictl and WSO2 API-M from here onwards, since we already introduced those in the first paragraph. Let's change the other occurences in this page as well.